satpy/composites/__init__.py

```diff
@@ -76,8 +73,7 @@ def check_times(projectables):
         # Is there a more gracious way to handle this ?
         if np.max(times) - np.min(times) > np.timedelta64(1, 's'):
             raise IncompatibleTimes
-        else:
-            mid_time = (np.max(times) - np.min(times)) / 2 + np.min(times)
+        mid_time = (np.max(times) - np.min(times)) / 2 + np.min(times)
```

Comment:
This line is just an indentation change. I'd prefer to leave any changes to this logic to another PR. This change was purely to make CodeFactor happy.

I think `times` is any `times` coordinate variable.

Reply:
yes, I gathered this was just cosmetic, but it got me thinking, sorry :) If times can be large, it might be worth to cache the min and max... otherwise, just ignore this.